examples/fluid_incompressible/benchmark/BrinkmanChannel2D/TRT/multi_f0.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+for F0 in 0.1 0.5 1 5 10 50 100
+do
+        # Write the value of F0 to args.lua
+        echo "F0 = $F0" > args.lua
```

Comment:
```suggestion
```

If you obtain the environment variable in `params.lua` (see the comment there), you do not need such an `args.lua` file at all.